pkg/cmd/triggertemplate/list.go

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package triggertemplate
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/helper/validate"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	"github.com/tektoncd/triggers/pkg/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	emptyMsg = "No triggertemplates found"
+)
+
+func listCommand(p cli.Params) *cobra.Command {
+	f := cliopts.NewPrintFlags("list")
+
+	eg := `
+# List all triggertemplates in namespace 'bar'
+tkn trigger list -n bar
+`
+
+	c := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "Lists triggertemplates in a namespace",
+		Annotations: map[string]string{
+			"commandType": "main",
+		},
+		Example: eg,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := validate.NamespaceExists(p); err != nil {
+				return err
+			}
+
+			cs, err := p.Clients()
+			if err != nil {
+				return err
+			}
+
+			tts, err := list(cs.Triggers, p.Namespace())
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to list triggertemplates from %s namespace \n", p.Namespace())
+				return err
```

Comment:
Maybe just return the error here instead of using `os.Stderr`?

```go
return fmt.Errorf("failed to list triggertemplates from %s namespace \n", p.Namespace())
```